README.md

```diff
@@ -16,7 +16,7 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
 
 Information about Maestro [README.md](src/Maestro/Readme.md)
 
-Please follow our [Validation Process](https://github.com/dotnet/core-eng/blob/master/Documentation/Validation/ValidationProcess.md) for including tests to go with new or changed functionality. 
+Please follow our [Validation Process](https://github.com/dotnet/core-eng/blob/master/Documentation/Validation/README.md) for including tests to go with new or changed functionality.
```

Comment:
Where would be a better place to put then? Since this is a public repo, and the build and deployment is private, not sure where else the info should go.

Reply:
I don't have a solution, hence why I just vented and I don't think it should block the PR.